.yamato/com.unity.ml-agents-coverage.yml

```diff
@@ -0,0 +1,65 @@
+test_editors:
+    - version: 2019.4
```

Comment:
Any reason we need to use 2019.4 here? Some of the extensions code needs 2020+ for ArticulationBodies.

Reply:
i can bump it there.  Didn't know 2020 was needed.